core/templates/pages/learner-dashboard-page/community-lessons-tab.component.ts

```diff
@@ -49,20 +50,28 @@ export class CommunityLessonsTabComponent {
   noPlaylistActivity: boolean;
   totalIncompleteLessonsList: (
     LearnerExplorationSummary | CollectionSummary)[] = [];
+
```

Comment:
Ditto*

Reply:
Resolving these too as they follow the similar pattern as stated in the above thread.